.circleci/config.yml

```diff
@@ -23,7 +26,7 @@ jobs:
     steps:
       - checkout
       - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
-          | python3 - --version 1.1.0"
+          | python3 - --version 1.1.4"
```

Comment:
Can the version be specified at the top of the file?

```
parameters:
  poetry_version:
    type: string
    default: "1.1.4"
```

Then wherever you use it you can do 

```
... | python3 - --version << pipeline.parameters.poetry_version >>
```